app/src/angular/controllers/member/dashboard/main-controller.js

```diff
@@ -58,6 +58,49 @@ function MemberDashboardMainController($rootScope, $scope, $interval, $log, $q,
 
         $scope.pieChart.draw();
     }
+    $scope.publicKeyHex = $rootScope.wallet.getPublicKeyHex();
+    $scope.transactionActivityIsSynced = function() {
+        let statuses = $rootScope.walletActivityStatuses;
+        let isInProgress = false;  
+        if (statuses) {
+            Object.keys(statuses).forEach(key => {
+                if (statuses[key] == false) {
+                    isInProgress = true;
+                }
+            });
+        }
+        return !isInProgress;
+    }
+
+    $scope.setTransactionAtivity = () => {
```

Comment:
i see this is not used in template.. remove it from **$scope**

Reply:
It's used in template(main.pug) 
div(ng-if="transactionActivityIsSynced", ng-init="setTransactionAtivity()")